tests/test_handle_files.py

```diff
@@ -0,0 +1,53 @@
+import unittest
+import tempfile
+import shutil
+from os.path import abspath, dirname, join
+
+from kodi_addon_checker.handle_files import find_file
+from kodi_addon_checker.handle_files import create_file_index
+from kodi_addon_checker.handle_files import find_file_recursive as FFR
+
+HERE = abspath(dirname(__file__))
+
+
+class TestFindFile(unittest.TestCase):
+
+    def setUp(self):
+        self.path = tempfile.mkdtemp()
+        self.file = join(self.path, "addon.txt")
+
+    def test_find_file_is_none(self):
+        self.assertIsNone(find_file("randomname", self.path))
+
+    def test_find_file_is_not_none(self):
+        with open(self.file, "w+"):
```

Comment:
Why does this need this line? Why does the function above not need this?

Reply:
This creates a new file on a given path so when tests are run the file can be found on a given location.